docs/platform/porting/graphics-and-ui.md

```diff
@@ -675,9 +738,10 @@ The Emulator YAGL (OpenGL&reg; ES / EGL&trade; driver for the emulator) is imple
 
 The following commit explains how to port the driver with `libtpl-egl` from the traditional drm-based driver:
 
-- Porting YAGL to the Tizen platform [https://review.tizen.org/gerrit/#/c/67921/](https://review.tizen.org/gerrit/#/c/67921/)
+- Porting YAGL to the Tizen platform [https://review.tizen.org/gerrit/c/platform/adaptation/emulator/emulator-yagl/+/67921](https://review.tizen.org/gerrit/c/platform/adaptation/emulator/emulator-yagl/+/67921)
+- Porting MESA to the Tizen platform [https://review.tizen.org/gerrit/c/platform/upstream/mesa/+/228724](https://review.tizen.org/gerrit/c/platform/upstream/mesa/+/228724)
 
-### Testing and Verifying the OpenGL&reg; ES Driver
+### Testing and Verifying OpenGL&reg; ES Driver
```

Comment:
@Samsung/tizen-docs-maintainer @Reni-90 

Khronos doesn't use HTML Symbols for headings recently: https://www.khronos.org/opengles/

I appreciate if you find out HTML Symbols in headings, please remove them or leave a comment to let authors remove them.

Reply:
Thank you very much for pointing this out.